### DISCUSSION TITLE
Spell Check Error!!!

### DISCUSSION THREAD

**Discussion by SW15999** [streetsidesoftware/vscode-spell-checker#2626] [Q&A]
Hello,When I spell "Behaviour",the plug tell me wrong,I should be changed to "Behavior",but the Unity API name is "MonoBehaviour".

**Comment by Jason3S** [comment]
`Behaviour` is the British spelling of `Behavior`. You can enable British spellings by setting the `language` preference to `en,en-GB`.

![image](https://user-images.githubusercontent.com/3740137/231671897-391ef120-5be8-4837-996d-7234351f391d.png)

**Reply by SW15999** [reply]
Thank you for your answer.
